solarized-dark-grey.css

```diff
@@ -1,1706 +1,549 @@
 /*
-* Solarized dark grey colors
-*
-*   background0  #141414
-*   background1  #212121
-*   background2  #2e2e2e
-*   background3  #3a3a3a
-*
-* Solarized dark base colors
-*
-*   base03       #002b36
-*   base02       #073642
-*   base01       #586e75
-*   base00       #657b83
-*   base0        #839496
-*   base1        #93a1a1
-*   base2        #eee8d5
-*   base3        #fdf6e3
-*
-* Solarized accent colors
-*
-*   yellow       #b58900
-*   orange       #cb4b16
-*   red          #dc322f
-*   magenta      #d33682
-*   violet       #6c71c4
-*   blue         #268bd2
-*   cyan         #2aa198
-*   green        #859900
+  Solarized Dark Grey - https://github.com/ristomatti/node-red-contrib-theme-solarized-dark-grey
 */
```

Comment:
Please don't delete this comment. It's a reminder to myself when editing. The plan originally was to process this using SCSS in order to use these as variables but I never got to that point.

Reply:
Fixing that.

Reply:
> The plan originally was to process this using SCSS in order to use these as variables

👆🏼 IMHO this is the best option. Much safer than using the code in this PR.